test/connectors/CodeLinkConnectorISpec.scala

```diff
@@ -19,7 +19,7 @@ package connectors
 import config.AppConfig
 import org.mockito.Mockito.{reset, when}
 
-class CodeLinkConnectorSpec extends ConnectorSpec {
+class CodeLinkConnectorISpec extends ConnectorISpec {
```

Comment:
Why rename to ISpec? This isn't really an integration test, its part of the unit tests.

Reply:
But it extends ConnectorISpec which in turn extends `GuiceOneAppPerSuite` and `WiremockTestServer` which are used for ITs